tests/suites/sidecar/sidecar.sh

```diff
@@ -64,3 +64,29 @@ check_snappass() {
 		sleep 5
 	done
 }
+
+test_pebble_notices() {
+	echo
+
+	# Ensure that a valid Juju controller exists
+	model_name="controller-model-sidecar"
+	file="${TEST_DIR}/test-${model_name}.log"
+	ensure "${model_name}" "${file}"
+
+	# Deploy Pebble Notices test application
+	# TODO(benhoyt): upload charm to Charmcraft
+	juju deploy /home/ben/w/juju/testcharms/charms/pebble-notices/juju-qa-pebble-notices_ubuntu-22.04-amd64.charm --resource redis-image=redis
```

Comment:
This will fail CI. Can we upload to charmhub first and deploy from there.

Reply:
Uploaded: https://charmhub.io/juju-qa-pebble-notices -- I've also asked for this to be changed from owner "Ben Hoyt" to "Juju Engineering" so any of you/us can upload it if necessary (https://discourse.charmhub.io/t/juju-qa-pebble-notices-change-of-ownership/12624).

Reply:
Fixed.